src/libraries/System.Console/src/System/Console.cs

```diff
@@ -277,14 +277,17 @@ static StrongBox<bool> EnsureInitialized()
         public static int CursorSize
         {
             get { return ConsolePal.CursorSize; }
+            [MinimumOSPlatformAttribute("windows7.0")]
```

Comment:
```suggestion
            [MinimumOSPlatform("windows7.0")]
```

In the `src` files throughout, I recommend dropping "Attribute" from all of the attributes and using `using System.Runtime.Versioning` statements as well; that will keep the attributes very succinct.

The `ref` files of course have to be left verbose.

Reply:
Let's not worry about this for today (Preview 8), but we can make the change after Preview 8 in a follow-up PR.

Reply:
It has low chance of passing CI and getting in by 3PM span time, so it should be fine to take an extra few minutes to fix the cosmetic issues too...

Reply:
done...